contribute.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Mozilla IRC iplimit",
+  "description": "Self-service tool for adding IRC exceptions for one's IP address",
+  "repository": {
+    "url": "https://github.com/mozilla/irc-iplimit",
+    "license": "MPL2"
+  },
+  "keywords": [
+    "Python",
+    "Flask",
+    "JSON"
+  ],
+	"participate": {
+		"home": "https://iplimit.irc.mozilla.org/",
+    "irc": "ircs://irc.mozilla.org",
```

Comment:
`ircs://` alone is not sufficient to direct clients to SSL port 6697. Please use `ircs://irc.mozilla.org:6697` here. (Note that while RFC 7194 defines `ircs-u` as the correct protocol for port 6697, not even Firefox recognizes that.)